app/geo/city.component.html

```diff
@@ -0,0 +1,4 @@
+<th scope="row">{{city.id}}</th>
```

Comment:
Geo folder - could you please create separate folders for geo.component and city.component inside geo folder?